main/src/library/RedBlackTree.flix

```diff
@@ -278,7 +279,7 @@ namespace RedBlackTree {
     /// Returns `None` if `tree` is the empty tree.
     ///
     pub def reduceRight(f: (k, v, k, v) -> (k, v) & ef, tree: RedBlackTree[k, v]): Option[(k, v)] & ef =
-        foldRight((k1, v1, x) -> match x {
+        foldRight((k1, v1, k) -> match k() {
```

Comment:
@magnus-madsen `k` does not work well with `RedBlackTree`, since the key-value pair is often `(k, v)`. Would you still prefer `k` as the accumulator? Or should I call it something else in these cases?

Reply:
Good point. Let's call it c in this file.